README.md

```diff
@@ -77,7 +82,10 @@ Belle is friends with Mrs. Potts
 Write your code here...
 
 ```ruby
-# code here
+
+friends.each do |friends|
+  puts "Belle is friends with #{friends}"
+end
 ```
```

Comment:
correct but definitely make sure not to have conflicting variables names. `|friends|` should probably be `|friend|` since its value will be an item in `friends`.